documents/TestSuite/pbrlib/edf/add_edf.mtlx

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="add_edf_test" type="">
```

Comment:
Would it be worthwhile to remove these empty type strings for clarity?

Reply:
Good catch, I will clean this up.